e2e_support/loom.yaml

```diff
@@ -1,6 +1,7 @@
 ChainID: 'default'
 QueryServerHost: "tcp://0.0.0.0:9999"
 RegistryVersion: 2
+ReceiptsVersion: 2
```

Comment:
We probably need to add support for multiple node configurations so we can run a node with different configs.

Reply:
On this PR ? or can I create an issue to accomplish this later ?

Reply:
Later